tests/production_restart.py

```diff
@@ -18,9 +18,10 @@
 # Once everything has been confirmed to be working correctly and finality is advancing, cleanly shut down the producer
 # defproducera node but keep the finalizer node of defproducerb running.
 #
-# Then change the finalizer policy (e.g. switch the order of the two finalizers) to get the nodes into a state where
-# they have a pending finalizer policy. At that point restart the producer node defproducera (with stale production
-# enabled so it produces blocks again).
+# Then change the finalizer policy using an unconfigured key in node defproducera to gurantee
```

Comment:
Typo: `gurantee`.

Also, the description above (while from the original issue) is confusing because both nodes have a producer and finalizer each. The description makes it seem like the first node is only a producer node.

Reply:
Thanks.